src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationResource.kt

```diff
@@ -1054,6 +1054,68 @@ class AdjudicationResource(
   ): UpdateHearingResultAwardResponses =
     adjudicationService.updateCreateAndDeleteHearingResultAwards(adjudicationNumber, chargeSequence, requests)
 
+  @PreAuthorize("hasRole('ROLE_NOMIS_ADJUDICATIONS')")
+  @PutMapping("/adjudications/adjudication-number/{adjudicationNumber}/charge/{chargeSequence}/quash")
+  @Operation(
+    summary = "updates adjudication charge outcome and awards to quashed",
+    description = "Latest a hearing result is updated to quashed along with all awards associated with this charge. Requires ROLE_NOMIS_ADJUDICATIONS",
```

Comment:
Latest? Is there a typo here somewhere, the sentence doesnt seem to make sense 🤔

Reply:
No, makes no sense at all - I'll do a new PR